$NetBSD: patch-qt4_pinentryconfirm_moc,v 1.1 2014/12/10 10:24:21 mef Exp $

Although, this is said generated file, it seems to me the path is 
wrong. (only pinentry-qt4 needs this patch)

--- qt4/pinentryconfirm.moc~	2014-12-10 17:40:26.000000000 +0900
+++ qt4/pinentryconfirm.moc	2014-12-10 17:41:41.000000000 +0900
@@ -6,7 +6,7 @@
 ** WARNING! All changes made in this file will be lost!
 *****************************************************************************/
 
-#include "../../../s/pinentry/qt4/pinentryconfirm.h"
+#include "../qt4/pinentryconfirm.h"
 #if !defined(Q_MOC_OUTPUT_REVISION)
 #error "The header file 'pinentryconfirm.h' doesn't include <QObject>."
 #elif Q_MOC_OUTPUT_REVISION != 63
